cove_360/lib/threesixtygiving.py

```diff
@@ -989,6 +1174,11 @@ def process(self, grant, path_prefix):
         OrganizationIdLooksInvalid,
         MoreThanOneFundingOrg,
         LooksLikeEmail,
+        ImpossibleDates,
```

Comment:
While I love this file as an interview question, if you did fancy refactoring it into a directory full of includes (and maybe even some fancy autofunc stuff, if Python lets you do that?) then I wouldn't object. But no worries if you don't fancy doing that right now.